### PR TITLE
Update landing page

### PR DESCRIPTION
**Pull request by jonparrott**
* Use friendlier language
* Be more authoritative
* Briefly explain the various types of documentation here.

### CODE REVIEW

**Comment by jonparrott** [comment]
Towards #317

**Comment by ncoghlan** [comment]
Taking a closer look at the build issue, it's just the reminder to update the reference set of deep links.

File: source/index.rst [hunk 1]
```diff
@@ -2,19 +2,37 @@
 Python Packaging User Guide
 ===========================
 
-The "Python Packaging User Guide" (PyPUG) aims to be the authoritative resource
-on how to package, publish, and install Python projects using current tools.
+Welcome to the *Python Packaging User Guide*. This guide provides documentation
+on how to distribute and install Python packages.
```
Comment (ddbeck) [inline source/index.rst]: The revised text here seems to be a lot less clear about what distinguishes this guide from, say, the setuptools or pip docs. If this text is going to change, I'd like to see it be even more reader focused. How about something like `Welcome to the *Python Packaging User Guide*, a collection of tutorials and references to help you distribute and install Python packages.` Or something like that. We can workshop it a bit. 😃

File: source/index.rst [hunk 2]
```diff
@@ -2,19 +2,37 @@
...
+
+Beyond our :doc:`tutorials/index`, this guide has several other resources:
+
+* :doc:`guides/index` that walk you through a specific task.
+* :doc:`discussions/index` that provide comprehensive documentation on specific topics.
+* :doc:`specifications/index` for detailed information about packaging specifications.
+
+Additionally, there is a list of :doc:`other projects <key_projects>` maintained by members of the Python Packaging Authority.
+
+Index
```
Comment (ddbeck) [inline source/index.rst]: `Index` should be `Contents`

File: source/index.rst [hunk 3]
```diff
@@ -2,19 +2,37 @@
...
+happily accept any :doc:`contributions and feedback <contribute>`. :)
 
-This guide is maintained on `github
-<https://github.com/pypa/python-packaging-user-guide>`_.
+.. _GitHub: https://github.com/pypa/python-packaging-user-guide>
+.. _Python Packaging Authority: https://pypa.io
 
-.. _docs.python.org: http://docs.python.org
+Get started
+===========
```
Comment (ddbeck) [inline source/index.rst]: This (and the following headings) ought to be level 2.

File: source/index.rst [hunk 4]
```diff
@@ -2,19 +2,37 @@
...
 
-The "Python Packaging User Guide" (PyPUG) aims to be the authoritative resource
-on how to package, publish, and install Python projects using current tools.
+Welcome to the *Python Packaging User Guide*. This guide provides documentation
+on how to distribute and install Python packages.
 
-To follow the development of Python packaging, see the `Python
-Packaging Authority <https://www.pypa.io>`_.
+This guide is maintained on `GitHub`_ by the `Python Packaging Authority`_. We
+happily accept any :doc:`contributions and feedback <contribute>`. :)
```
Comment (ddbeck) [inline source/index.rst]: Gratuitous personal preference: change `:)` to 😄

File: source/index.rst [hunk 5]
```diff
@@ -2,19 +2,37 @@
...
 
-.. _docs.python.org: http://docs.python.org
+Get started
+===========
 
+If you want to learn how to install packages, see the :doc:`tutorial on installing packages <tutorials/installing-packages>`.
+
+If you want to learn how to package and distribute your projects, see the :doc:`tutorial on packaging and distributing <tutorials/distributing-packages>`
+
+More resources
```
Comment (ddbeck) [inline source/index.rst]: If the ToC below is going to show more depth, then I don't think this section is necessary. Skip right to the ToC.

File: source/index.rst (hunk 3 above)
Comment (jonparrott) [inline source/index.rst]: They are.

File: source/index.rst (hunk 2 above)
Comment (jonparrott) [inline source/index.rst]: Done.

File: source/index.rst (hunk 5 above)
Comment (jonparrott) [inline source/index.rst]: I actually want to try to hide the ToC on this page, gotta tweak the theme a bit to get that to work.

File: source/index.rst (hunk 4 above)
Comment (jonparrott) [inline source/index.rst]: Done.

File: source/index.rst (hunk 1 above)
Comment (jonparrott) [inline source/index.rst]: I like it, updated. :)

File: source/index.rst (hunk 5 above)
Comment (jonparrott) [inline source/index.rst]: Okay, toc hidden, you can check it out here: http://temp.theadora.io/pypug-326/index.html

Comment (ddbeck) [inline source/index.rst]: OK. I'm not in love with not having the full ToC visible anywhere, but I'll cope. A couple more comments coming up, in this case.

Comment (ddbeck) [inline source/index.rst]: Maybe retitle this section as "Learn more" for parallelism with "Get started"

File: source/index.rst [hunk 6]
```diff
@@ -24,3 +14,31 @@ This guide is maintained on `github
...
+==============
+
+Beyond our :doc:`tutorials/index`, this guide has several other resources:
+
+* :doc:`guides/index` that walk you through a specific task.
+* :doc:`discussions/index` that provide comprehensive documentation on specific topics.
+* :doc:`specifications/index` for detailed information about packaging specifications.
+
```
Comment (ddbeck) [inline source/index.rst]: If these bullets are going to signpost the rest of the ToC, let's be a little be a little more forthcoming about what these sections contain. How about something like this:

```
* :doc:`guides/index` for walk throughs, such as :doc:`guides/installing-using-linux-tools` or :doc:`guides/packaging-binary-extensions`
* :doc:`discussions/index` for in-depth references on topics such as :doc:`discussions/deploying-python-applications` or :doc:`discussions/pip-vs-easy-install`
* :doc:`specifications/index` for packaging interoperability specifications
```

File: source/index.rst [hunk 7]
```diff
@@ -24,3 +14,31 @@ This guide is maintained on `github
...
+Additionally, there is a list of :doc:`other projects <key_projects>` maintained by members of the Python Packaging Authority.
```
Comment (ddbeck) [inline source/index.rst]: `members of the Python Packaging Authority` -> `the Python Packaging Authority`

Well, probably? I'm not sure the distinction between members and the PyPA itself is meaningful. I assume it isn't and figure this change would be consistent with the other line about the PyPA above.

Comment (jonparrott) [inline source/index.rst]: I think it's important here to keep the distinction, some of these projects are rather independent.

File: source/index.rst (hunk 5 above)
Comment (jonparrott) [inline source/index.rst]: done.

File: source/index.rst (hunk 6 above)
Comment (jonparrott) [inline source/index.rst]: Good suggestion, done.

File: source/index.rst (hunk 7 above)
Comment (ddbeck) [inline source/index.rst]: OK, sounds good.

File: source/index.rst (hunk 6 above)
Comment (ddbeck) [inline source/index.rst]: One minor thing: can we lose the periods on the ends of the bullets? It's a list of things, not really completely sentences

Comment (jonparrott) [inline source/index.rst]: Done.

**Comment by jonparrott** [comment]
Thanks for the review, @ddbeck!

**Comment by ncoghlan** [comment]
I like the update, but it turns out that the paragraph based structure of the `Get started` section made the tutorial links less obvious to me than the bullet points in the `Learn More` section.

What do you think about laying out that section something like:

## Get Started

Essential tools and concepts for working with the Python Packaging ecosystem are covered in our Tutorials section:

* to learn how to install packages, see the tutorial on installing packages.
* to learn how to package and distribute your projects, see the tutorial on packaging and distributing

That way, all 5 of the key links on the page (installation tutorial, publishing tutorial, guides section, discussion section, specification section), would all be highlighted as bullet points.

**Comment by ncoghlan** [comment]
(PR incoming, as I originally didn't write one because I wasn't sure how I thought the section *should* look, but articulating the question clarified what I actually wanted)

**Comment by jonparrott** [comment]
Sounds good. Looking forward to the PR. :)

**Comment by ncoghlan** [comment]
@jonparrott already reviewed it, but for future reference: https://github.com/pypa/python-packaging-user-guide/pull/327 :)

**Pull request merged**